lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1111,6 +1111,12 @@ LLVMToSPIRVBase::getLoopControl(const BranchInst *Branch,
 
   size_t LoopControl = spv::LoopControlMaskNone;
   std::vector<std::pair<SPIRVWord, SPIRVWord>> ParametersToSort;
+  // We need to create these variables here because it might be the case when
+  // metadata is not defined for one or two of these parameters and in this
+  // case we need to store "-1" value
```

Comment:
```suggestion
  // If only a subset of loop count parameters is defined in metadata
  // then undefined ones should have a default value -1 in SPIR-V.
  // Preset all loop count parameters with the default value.
```

Reply:
Changed in [a12e1a0](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1062/commits/a12e1a08a62ea0b5993037c27caf8aa39f72231b)